README.md

```diff
@@ -51,219 +51,219 @@ go run schemagen.go -admin-facades -facade-group client ./schemas/client-schemas
+| Action               | Run                             |                                                                                              |
+| Action               | RunOnAllMachines                |                                                                                              |
+| Action               | WatchActionsProgress            |                                                                                              |
+| Admin                | Login                           |                                                                                              |
+| Admin                | RedirectInfo                    |                                                                                              |
+| AllModelWatcher      | Next                            |                                                                                              |
+| AllModelWatcher      | Stop                            |                                                                                              |
+| AllWatcher           | Next                            |                                                                                              |
+| AllWatcher           | Stop                            |                                                                                              |
+| Annotations          | Get                             |                                                                                              |
+| Annotations          | Set                             |                                                                                              |
+| Application          | AddRelation                     | `POST /integrate`                                                                            |
+| Application          | AddUnits                        |                                                                                              |
+| Application          | ApplicationsInfo                | `GET /model/{model}/application/{application}`                                                |
+| Application          | CharmConfig                     |                                                                                              |
+| Application          | CharmRelations                  |                                                                                              |
+| Application          | Consume                         |                                                                                              |
+| Application          | Deploy                          |                                                                                              |
+| Application          | DeployFromRepository            | `POST /model/{model}/application/{application}/deploy&dry-run=false&trust=false` |
```

Comment:
@cderici this needs updating to reflect the changes in the actual openapi spec :)